dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewManagerScope.kt

```diff
@@ -78,8 +78,13 @@ internal class RumViewManagerScope(
     @WorkerThread
     private fun startApplicationLaunchView(event: RumRawEvent, writer: DataWriter<Any>) {
         val applicationLaunchViewTimeNs = appStartTimeProvider.appStartTimeNs
+        // appStartTimeNs is based on elapsed system time. Timestamp needs to be currentMs.
```

Comment:
I find this description confusing. `timestamp` doesn't need to be _current_ time.

Basically `Time#timestamp` and `Time#nanoTime` are pointing to the same instant (moment of the time), while having different base - `timestamp` is based on the Unix epoch start, `nanoTime` is based on the JVM start time (for Android I think it is efficiently the process start/fork time). 

So I guess the comment wanted to say that `timestamp` should have the same base as `System.currentTimeMillis` (just the shortened abbreviation is used)?

So what we are doing below is first calculating the different between 2 instant representations, which is `event.eventTime.timestamp - TimeUnit.NANOSECONDS.toMillis(event.eventTime.nanoTime)` to find out when actually process has started in terms of epoch time, and then apply the time since this process start (which is `applicationLaunchViewTimeNs`) to find when even happened in the epoch time format.

So I would suggest for the clarity to introduce the variable `processStartNs` with the description what is this (also maybe we need to keep calculation in the ns and convert to the ms at the very end, better probably precision-wise (although 10^6 different between ns and ms will erase this precision gain anyway)), because I think without it the expression below is not quite clear.

Reply:
👍 I'll take another stab at it.